Sources/SpeziOpenAI/MessageView.swift

```diff
@@ -67,27 +63,27 @@ public struct MessageView: View {
         }
     }
     
+    public enum Defaults {
+        static let hideMessagesWithRoles: Set<Chat.Role> = [.system, .function]
+    }
```

Comment:
You can address the SwiftLint errors by moving this nested type at the beginning of the enclosing type. You can find out more about the rule here: https://realm.github.io/SwiftLint/type_contents_order.html